test-infra/no-cluster/stunnel/pki/private.pem

```diff
@@ -0,0 +1,48 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
So we made this exact same mistake in redis-py, which IMHO is tech debt. When I originally committed it, I thought it smarter to commit junk certs (and a README), but it led to fallout. Since they're generated  via [this script](https://github.com/redis/redis/blob/unstable/utils/gen-test-certs.sh) from redis, maybe we should just fetch it, and run it first? We'd need a modification given the last line, but the only requirement is openssl.

Reply:
I agree, let's integrate that. In another PR?